relayer/relays/beacon/header/header.go

```diff
@@ -265,7 +265,10 @@ func (h *Header) SyncHeaders(ctx context.Context) error {
 	}).Info("starting to back-fill headers")
 
 	blockRoot := common.HexToHash(finalizedHeader.FinalizedHeader.ParentRoot.Hex())
-	prevSyncAggregate := finalizedHeader.SyncAggregate
```

Comment:
This sync aggregate is actually the `attested_header` sync witness, not the `finalized_header`'s sync witness.